src/core/ports/CodeGouvApi.ts

```diff
@@ -0,0 +1,72 @@
+export type CodeGouvApi = {
+    // Main data
+	getRepositories: {
+        (): Promise<Repository[]>;
+    },
+	getDependencies: {
+		(): Promise<Dependency[]>;
+	},
```

Comment:
```suggestion
	getDependencies: (): Promise<Dependency[]>;
```
On n'exprime la functino comme un objet que quand on à d'autre methodes sur l'objet (genre `clear()`)

Reply:
D'ailleur je dirais que toutes les functions devrais être mémoizer ici

Reply:
> D'ailleur je dirais que toutes les functions devrais être mémoizer ici

J'y réfléchis depuis hier mais je ne suis pas sûre d'être d'accord. De manière générale, j'ai même l'impression que la memoization de toutes les fonctions n'est pas pertinente. Par exemple, la liste des repos ne sera récupérée qu'une seule fois dans toute l'app, donc le cache ne sera jamais atteint. Il serait sûrement plus pertinent de gérer un cache dans le state de l'application (avec des paramètres de filtre et de recherche par exemple). Qu'en penses-tu @mineraux ?

En conclusion, je vais finalement retirer complétement la memoization pour le moment. :)

Reply:
Je suis d'accord que la mémoization doit se faire de façon pertinente.
On peut faire une première version avec un cache dans le state de l'application, on verra si on a des problèmes de performance. Je ne pense pas que ça sera le cas mais même si ça arrive, ce n'est pas le genre de refacto qui prend énormément de temps si on se laisse du temps de recette.